### ISSUE TITLE
getTopic() is protected

### ISSUE THREAD

**Issue by AppWerft** [eclipse-paho/paho.mqtt.java#214]
- [x] Bug exists Release Version 1.0.2 ( Master Branch)
- [ ] Bug exists in Snapshot Version 1.0.3 (Develop Branch)

I try to instantiate a new MqttClient in my package. The implemented method _getTopic(String)_ is protected in jar (org.eclipse.paho.client.mqttv3-1.0.2.jar) and I cannot call.
This is error on console:

```
  [javac] /Users/fuerst/Documents/APPC_WORKSPACE/Ti.MQTTclient/android/src/de/appwerft/mqtt/MqttClient.java:477: error: getTopic(String) has protected access in MqttAsyncClient
```

**Comment by jpwsutton** [comment]
@AppWerft Could you confirm this in the new released version (V1.1.0)
http://search.maven.org/#artifactdetails%7Corg.eclipse.paho%7Corg.eclipse.paho.client.mqttv3%7C1.1.0%7Ceclipse-plugin

**Comment by jpwsutton** [comment]
@AppWerft is there any reason why you need to use getTopic? Could you not just use the publish method which will effectively do the same thing before publishing?
